libipl/p10/ipl0.C

```diff
@@ -344,6 +344,55 @@ static void process_guard_records()
 	}
 }
 
+//@Brief Function will check if the FCO state is set for hardware units
+// consumed by sbe or not. If FCO override bit is set for such hardware
+// units, need to mark present and functional state as set in devicetree
+// during the boot.
+static bool apply_fco_override(void)
+{
+	std::array<const char*, 8> mProcChild =
+		{"core", "pauc", "pau", "iohs", "mc", "chiplet", "pec", "fc"};
```

Comment:
This feature enabled to support sbe required targets.

Reply:
The only targets that are explicitly required right now are core and fc.  However, if you just blindly look at all targets you would be more future proof in case a new layer is added to the target hierarchy going forward.  There is no downside to looking at other types beyond the minor computation time.